java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -667,4 +668,47 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Long> zlexcount(String key, LexRange minLex, LexRange maxLex);
+
+    /**
+     * Computes the union of sorted sets given by the specified <code>keys</code>, and stores the
+     * result in <code>destination</code>. If <code>destination</code> already exists, it is
+     * overwritten. Otherwise, a new sorted set will be created.
+     *
+     * @see <a href="https://redis.io/commands/zunionstore/">redis.io</a> for more details.
+     * @param destination The key of the destination sorted set.
+     * @param keys The keys of sorted sets.
+     * @param options Weight and Aggregate options.
+     * @return The number of elements in the resulting sorted set.
+     * @example
+     *     <pre>{@code
+     * WeightAggregateOptions options =
+     *     WeightAggregateOptions.builder()
+     *             .aggregate(Aggregate.MAX)
+     *             .weights(List.of(1.0, 2.0))
```

Comment:
I think I'd prefer the example to use: 
```
     *             .weight(1.0)
     *             .weight(2.0)
```

Reply:
Why?

Reply:
1. we get to take advantage of chaining in the builder - which is the whole purpose of using a builder. 
2. fewer brackets (2 brackets in this case isn't bad... but 1 is easier on the eyes)
3. abstract away the "List" completely from the user
They are all minor... but this is the whole reason why we included the @Singular annotation